spec/index.html

```diff
@@ -153,1920 +183,1796 @@ <h2>Semantic Extensions and Entailment Regimes</h2>
+          <span class="space"></span>
+          <p>If <em>E</em> is a ground triple `s p o.`, then <em>I(E) = true</em> if<br>
+            <em>I(p)</em> is in <em>IP</em> and the pair &lt;<em>I(s)</em>, <em>I(o)</em>&gt;
+            is in <em>IEXT(I(p))</em><br>
+            otherwise <em>I(E) = false</em>.
+          </p>
+          <span class="space"></span>
+          <p>If <em>E</em> is a ground RDF graph, then <em>I(E) = false</em> if <em>I(E')</em> =
+            false for some triple <em>E'</em> in <em>E</em>, otherwise <em>I(E) = true</em>.
+          </p>
+        </div>
+      </section>
+      <p>If IL(E) is undefined for some literal <em>E</em> then <em>E</em> has no semantic value,
+        so any triple containing it will be false,
+        so any graph containing that triple will also be false.
+      </p>
+      <p>The final condition implies that the empty graph (the empty set of triples) is always true.</p>
+      <p>The sets IP and IR may overlap, indeed IP can be a subset of IR.
+        Because of the domain conditions on IEXT,
+        the referent of the subject and object of any true triple will be in IR;
+        so any IRI which occurs in a graph both as a predicate and as a subject or object
+        will denote something in the intersection of IP and IR.
+      </p>
+      <p><a>Semantic extensions</a> may impose further constraints upon interpretation mappings
+        by requiring some IRIs to denote in particular ways.
+        For example, D-interpretations, described below, require some IRIs,
+        understood as <a>identify</a>ing and referring to datatypes, to have a fixed referent.
+      </p>
+      <section id="blank_nodes">
+        <h3>Blank nodes</h3>
+        <p>Blank nodes are treated as simply indicating the existence of a thing,
+          without using an IRI to <a>identify</a> any particular thing.
+          This is not the same as assuming that the blank node indicates an 'unknown' IRI.
+        </p>
+        <p>Suppose <em>I</em> is a simple interpretation and A is a mapping from a set of blank nodes
+          to the universe IR of I.
+          Define the mapping <em>[I+A]</em> to be <em>I</em> on <a>names</a>,
+          and A on blank nodes on the set: <em>[I+A](x)=I(x)</em> when <em>x</em> is a <a>name</a>
+          and <em>[I+A](x)=A(x)</em> when <em>x</em> is a blank node;
+          and extend this mapping to triples and RDF graphs using the rules given above for ground graphs.
+          Then the semantic conditions for an RDF graph are:
+        </p>
+        <section class="notoc">
+          <h6 class="defcap">Semantic Condition for Blank Nodes</h6>
+          <div class="semantictable">
+            <p>If <em>E</em> is an RDF graph, then <em>I(E) = true</em> if [<em>I + A</em>](<em>E</em>) = 
+              true for some mapping <em>A</em> from the set of blank nodes in <em>E</em> to <em>IR</em>, 
+              otherwise <em>I(E) = false</em>.
```

Comment:
Again, highlight fail.
```suggestion
            <p>If <em>E</em> is an RDF graph and <em>[I + A](E) = true</em>
              for some mapping <em>A</em> from the set of blank nodes in
              <em>E</em> to <em>IR</em>, then <em>I(E) = true</em>; 
              otherwise, <em>I(E) = false</em>.
```